README.md

```diff
@@ -1,14 +1,25 @@
 # Firebase Auth with Google and React Context API   
+Quick links 
+[Set Up](#project-set-up)
+[Firebase API](firebase-api)
+[React useContext()](react-usecontext()) 
+[UserContext](managing-usercontext)
+[Auth State](auth-and-state)
+[Summary](#putting-it-all-together)
 
 This project will walk you through the steps to set up User Authentiaction via Google sign in using Firebase, React and React Context.  
 
 ## What is Firebase?
 
-Firebase is whats known as  a BAAS, or Back End As A Service, for creating mobile and web applications. It was originally an independent company founded in 2011.  In 2014, Google acquired the platformand it is now their flagship offering for app development. Its primary features include a Realtime noSQL database, and user authentication that all come out of the box and ready to connect to a front end application.  While using Firebase is very convenient in many ways it also limits the scope of a full stack applicaiton by limiting the way we structure our database or backened functionality.  It is great for smaller applicaitons that do not need a large back end layer of complexity or to bring in certain features (LIKE AUTHENTICATION)
+Firebase is whats known as  a BAAS, or Back End As A Service, for creating mobile and web applications. 
+
+It was originally an independent company founded in 2011.  In 2014, Google acquired the platformand it is now their flagship offering for app development. Its primary features include a Realtime noSQL database, and user authentication that all come out of the box and ready to connect to a front end application. 
+
+While using Firebase is very convenient in many ways it also limits the scope of a full stack applicaiton by limiting the way we structure our database or backened functionality.  It is great for smaller applicaitons that do not need a large back end layer of complexity or to bring in certain features (LIKE AUTHENTICATION)
```

Comment:
noSQL is not ideal for uses that have a lot of relational data. 

These days, it is not uncommon for an application to use multiple databases for different aspects of an app. For example, using firebase for authentication, but Postgres for storing product and order information.